setup.py

```diff
@@ -57,4 +57,5 @@ def get_long_description() -> str:
     url="https://github.com/ome/omero-cli-zarr/",
     setup_requires=["setuptools_scm==7.1.0"],
     use_scm_version={"write_to": "src/omero_zarr/_version.py"},
+    tests_require=["omero-py>=5.18.0", "pytest", "omero-rois"],
```

Comment:
Oops. I think we wanted to omit this omero-rois, right?

Reply:
I'm happy to remove it. Could remove the whole line as it's not being used? Unless it's useful to someone running the tests locally to know that omero-rois is required?